### ISSUE TITLE
DeepFlash Model

### ISSUE THREAD

**Issue by sepideh-srj** [jozech/flash-to-ambient#1]
Hi, 
I am trying to use the deep flash model, but the result is just the same as the input. Any idea what I'm doing wrong?

**Comment by jozech** [comment]
Hi,

I didn't upload the weights of the deep flash model to my drive. You may
train the model from scratch.

José

**Comment by sepideh-srj** [comment]
yes I am training from scratch but still it's not giving me anything at all.

**Comment by jozech** [comment]
Please, try to use "evalM_oneimg.py" with "--load_epoch".

**Comment by sepideh-srj** [comment]
Hi again,

I was wondering if you have the pretrained vgg for initializing the weights, can you share it with me?
I mean the pretrained vgg used for face recognition, not the one you trained on your data.
Thanks

**Comment by jozech** [comment]
Sorry, I don't have this information. Maybe you should contact the paper
author.
